client/apiversion_test.go

```diff
@@ -0,0 +1,99 @@
+package client_test
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+)
+
+type versionHandler struct {
+	authBody string
+	port     string
+}
+
+type MakeServiceURLTest struct {
+	serviceType string
+	version     string
+	parts       []string
+	success     bool
+	URL         string
+	err         string
+}
+
+func (s *localLiveSuite) makeServiceURLTests() []MakeServiceURLTest {
```

Comment:
Perhaps this would be best declared near the test cases which make use of it. Maybe move the MakeServiceURL tests to this file?